src/extension.ts

```diff
@@ -67,18 +76,22 @@ function spotifyAuthentication(spotify: typeof SpotifyWebApi) {
 			code = req.query.code;
 			spotify.authorizationCodeGrant(code).then(
 				data => {
+					accessToken = data.body['access_token'];
 					spotify.setAccessToken(data.body['access_token']);
     				spotify.setRefreshToken(data.body['refresh_token']);
 				},
 				err => {
 					vscode.window.showErrorMessage(`Error exchanging authorization code for access token: ${err}`);
 				}
 			);
+			
 			res.send('You may now close this tab.');
 			vscode.window.showInformationMessage("Successfully authenticated with Spotify.");
 			server.close();
 		}
 	});
+
+
 	server = app.listen(PORT);
 	vscode.env.openExternal(authUrl);
-}
+}
```

Comment:
Leaving a blank line at the end of a file is good form, for mostly trivial reasons but if you're wondering why that red circle is here, that's why.

```suggestion
}

```